.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<-- Start the PR description with some context for the change. -->
+
+
+<-- Make sure to delete the markdown comments and the below sections when squash merging -->
```

Comment:
Do you mean to say to complete the below sections before squash merging?

Reply:
No. I don't think we want anything from this template other than the context in the final merge commit, hence the comment to delete all of the `<-- -->` and checklist.

Reply:
Oh, I see. I think I normally just delete everything and leave the description on my PR to look at if someone wants context.

...Should I change that?

Reply:
In general, I recommend leaving the description of your PR as the body of your commit message, just because then people can get some initial context from calling "git log" or "git show <commit>" without having to go GitHub or Azure DevOps.

Reply:
Ok, that makes sense.